python/ray/remote_function.py

```diff
@@ -21,6 +21,32 @@
 logger = logging.getLogger(__name__)
 
 
+_task_launch_hook = None
+
+
+def tune_task_launch_hook(fn, resources):
+    if "_export_bundle" in fn.function_name:  # delete this later
```

Comment:
So when I tried this, `fn` can be `_export_bundle_xxx`, `objective` or `foo`.
How to make sure that we don't throw this error for `objective`? My test bench seems to be throwing it..

Reply:
Moved to the session runner.